.gitlab-ci.yml

```diff
@@ -234,6 +233,12 @@ windows32:
   except:
     - /^pr-.*$/
 
+pkg:dune-release:
+  <<: *dune-template
+  stage: test
```

Comment:
When putting stuff in the test stage we should use `dependencies: []` to avoid downloading the various built coqs.

Reply:
In fact let's put it in the dune-template as that seems simpler.

Reply:
Good point, thanks.

Reply:
> In fact let's put it in the dune-template as that seems simpler.

I've leave it like this, but note that Dune has no problem in reusing previous builds in an incremental way.

Reply:
>I've left it like this, but note that Dune has no problem in reusing previous builds in an incremental way.

Including across profiles?
And there's certainly no point in downloading the various `make` based builds.

Reply:
Builds live in "contexts", the only one is default but you for example could declare one for `bisect_pxx` etc... they are independent.

`(env ...)` profiles are supposed to be tracked in the dependency chain, so indeed if you change some stuff or profile it may trigger a rebuild. Note that this is still being developed AFAICT.

Reply:
> And there's certainly no point in downloading the various make based builds.

Sure, I was just mentioning in case in the future we need it. The dependency tracking is more useful for the developer tho.